operator/README.md

```diff
@@ -0,0 +1,151 @@
+|     |    |                            |                |      |                     |
+|     |    |   operator                 |                |      |                     |
+| +---+----+----------------------------+--------+       |      |                     |
+| |   |    |                            |        |       |      |                     |
+| |  ++----+----+                 +-----v----+   |       |      |                     |
+| |  |sidecar   <-----------------> teleport <---+-------+------>                     |
+| |  +----------+ Setup U/R       | operator |   |       |      |                     |
+| |             Create Identity   +----------+   |       |      |                     |
+| |                                              |       |      |                     |
+| |                                              |       |      |                     |
+| +----------------------------------------------+       |      +---------------------+
+|                                                        |
+|                                                        |
+|                                                        |
++--------------------------------------------------------+
+```
+
+## Running
+
+### Requirementes
+
+#### K8S cluster
+You can use `minikube` if you don't have a cluster yet.
+
+#### Operator's docker image
+You can obtain the docker image by pulling from `quay.io/gravitational/teleport`
+
+#### HELM chart from Teleport with the operator
+We are re-using the Teleport Cluster Helm chart but modifying to start the operator using the image above.
+
+This change is not in master, so you'll need to checkout the HELM charts from this branch
+`marco/plugins-teleport-operator-charts` (https://github.com/gravitational/teleport/pull/12144)
+
+#### Other tools
+We also need the following tools: `helm`, `kubectl` and `docker`
+
+### Running the operator
+
+Start `minikube` with `minikube start`.
+
+Set the `TELEPORT_PROJECT` to the full path to your teleport's project checked out at `marco/plugins-teleport-operator-charts`.
+
+Install the helm chart:
+```bash
+$ helm upgrade --install --create-namespace -n teleport-cluster \
+	--set clusterName=teleport-cluster.teleport-cluster.svc.cluster.local \
+	--set teleportVersionOverride="9.3.5" \
+	--set operator=true \
+	teleport-cluster ${TELEPORT_PROJECT}/examples/chart/teleport-cluster
+
+$ kubectl config set-context --current --namespace teleport-cluster
+
+```
+
+Now let's wait for the deployment to finish:
+`kubectl wait --for=condition=available deployment/teleport-cluster --timeout=2m`
+
+If it doesn't, check the errors.
+
+Now, we want access to two configuration tools using a Web UI: K8S UI and Teleport UI.
+
+First, let's create a tunnel using minikube: `minikube tunnel` (this command runs is foreground, open another terminal for the remaining commands).
+
+Now, let's create a new Teleport User and login in the web UI:
```

Comment:
`teleoperator` is the user we create to navigate the Teleport's webui
We can just create another user

This can also be from a previous version we had (tagged as 9.3.5)
Can you please test the same using the dev build?

```bash
minikube delete; minikube start;
export TELEPORT_PROJECT=<github.com/gravitation/teleport checked out at `marco/plugins-teleport-operator-charts`>
helm upgrade --install --create-namespace -n teleport-cluster \
	--set clusterName=teleport-cluster.teleport-cluster.svc.cluster.local \
	--set teleportVersionOverride="11.0.0-marco.3" \
	--set operator=true \
	teleport-cluster ${TELEPORT_PROJECT}/examples/chart/teleport-cluster
```
(You don't need to run `minikube delete` afterwards, this is just to make sure we clean the state of previous builds that might have created the wrong sidecar user)

Reply:
Yep, looks like running `minikube delte` and using the newer build fixes the issue, thanks!